plugins/node/instrumentation-undici/README.md

```diff
@@ -16,7 +16,7 @@ npm install --save @opentelemetry/instrumentation-undici
 
 ## Supported Versions
 
-- `undici@>=5.12.0`
+[`undici`](https://www.npmjs.com/package/undici) version `>=5.12.0`
```

Comment:
for `unidici` the instrumentation seems to not target a specific version range, but using `diagnosticChannel`. therefore I don't think we can say anything about the upper limit we support.